components/permit-holders/GuardianInformationCard.tsx

```diff
@@ -3,29 +3,28 @@ import PermitHolderInfoCard from '@components/internal/PermitHolderInfoCard'; //
 import { Guardian } from '@lib/graphql/types'; // Guardian type
 
 type GuardianInformationProps = {
-  guardian: Guardian;
+  guardian?: Guardian;
```

Comment:
```suggestion
  guardian: Guardian;
```
This prop should be mandatory. If an applicant doesn't have a guardian, we shouldn't render this card at all.